autotest/gdrivers/miramon_read.py

```diff
@@ -0,0 +1,1074 @@
+#!/usr/bin/env pytest
+###############################################################################
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  Test basic read support for all datatypes from a MiraMon file.
+# Author:   Abel Pau <a.pau@creaf.cat>
+#
+###############################################################################
+# Copyright (c) 2025, Abel Pau <a.pau@creaf.cat>
+#
+# SPDX-License-Identifier: MIT
+###############################################################################
+
+
+import math
+import os
+import struct
+
+import pytest
+
+from osgeo import gdal
+
```

Comment:
you can remove all ``@pytest.mark.require_driver("MiraMonRaster")`` if you just add

```suggestion

pytestmark = pytest.mark.require_driver("MiraMonRaster")

```